tools/cloud-build/babysit_tests.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
```

Comment:
LGTM except I think we need some minimal documentation. Just a few sentences that says the purpose of the tool. You could add it as description in argparse OR to `tools/cloud-build/README.md` OR docstring on Babysitter class.